### ISSUE TITLE
fix-request: Missing interface causing eslint incompatibilities in older versions

### ISSUE THREAD

**Issue by noherczeg** [estree/estree#271]
As per https://github.com/Azure/autorest.typescript/issues/1230#issuecomment-1007864414,

- @types/estree: 0.0.39
- @types/eslint: 8.2.2

Are (or can) cause build errors in certain projects setups at least. Also as per my comment, Adding the following interfaces to version `0.0.39` seems to fix the issue:

```
export interface ThisExpression extends BaseExpression {
  type: "ThisExpression";
}

export interface ImportExpression extends BaseExpression {
  type: "ImportExpression";
}
```

I have no idea what the process is to create a fix version for 0.0.39, but if anyone can help, I'm happy to contribute!

**Comment by JLHwung** [comment]
`@types/estree` is not maintained in this project. Please open an issue at https://github.com/DefinitelyTyped/DefinitelyTyped, thank you.

**Comment by noherczeg** [comment]
@ffMathy I checked the PR process explained here https://github.com/DefinitelyTyped/DefinitelyTyped#make-a-pull-request  and I got scared away, lol. Not to mention, the types I referenced in this issue are already there in the latest version. Therefore this whole issue is about some mismatch in different libraries and different versions of transitive dependencies. More then likely this mismatch is not an estree issue directly.
